core/config/config.go

```diff
@@ -66,7 +66,7 @@ func getConfigFiles() []string {
 
 func getResolver() FileResolver {
 	paths := []string{}
-	configDir := os.Getenv(GetEnvironmentPrefix() + configdir)
+	configDir := GetConfigurationPath()
```

Comment:
Don't name things `Getfoo()` -- that's a Java-ism and is not idiomatic. just `ConfigurationPath`, although that stutters, so consider just `Path()` or something.